packages/codemods/src/codeMods/mods/configMod/configMod.mod.ts

```
@@ -144,7 +144,7 @@ const configMod: CodeMod = {
       results.push(mod.run(file));
     }
     if (results.length === 0) {
-      return Err<ModResult, NoOp>({ logs: ['No runabble mods were found in the config'] });
```

Changed from `Err` to `Ok` because the log message is swallowed otherwise.